### DISCUSSION TITLE
Concourse CI build and pipeline logs

### DISCUSSION THREAD

**Discussion by Krish92na** [concourse/concourse#7648] [Help & Support]
I'm fairly new to the Concourse ethos, so I was hoping for some guidance from our community. Tried searching for other discussions that answered my query but couldn't find any.

Need to view the logs created when a build or pipeline is triggered, as in details around which user triggered it, what action was performed by the user. I'm unable to find the appropriate path where such logs reside.

Thanks in advance

**Comment by taylorsilva** [comment]
Have you seen this? https://concourse-ci.org/concourse-web.html#audit-logs
